homeassistant/components/downloader.py

```diff
@@ -109,14 +113,15 @@ def do_download():
 
                     # If file exist append a number.
                     # We test filename, filename_2..
-                    tries = 1
-                    final_path = path + ext
-                    while os.path.isfile(final_path):
-                        tries += 1
+                    if overwrite is False:
+                        tries = 1
+                        final_path = path + ext
+                        while os.path.isfile(final_path):
+                            tries += 1
 
-                        final_path = "{}_{}.{}".format(path, tries, ext)
+                            final_path = "{}_{}.{}".format(path, tries, ext)
 
-                    _LOGGER.info("%s -> %s", url, final_path)
+                        _LOGGER.info("%s -> %s", url, final_path)
```

Comment:
We still want to log which url we're downloading to which location, no matter if we renamed target file.